fri/test.py

```diff
@@ -17,8 +17,8 @@ def upload(files):
 # # *******
 
 # function to check build
-def build(dir, graphml, apikey):
-  url = "http://127.0.0.1:5000/build/"+dir+"?"+"fetch="+graphml+"&"+"apikey="+apikey

```

Comment:
Instead of removing the "&"+"apikey="+apikey, you should actually add this uniformly across all methods and test it.

Reply:
Understood